src/components/Levels/BobaShop.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+
+import b1 from '../../assets/BobaAssets/b1.svg';
+import b2 from '../../assets/BobaAssets/b2.svg';
+import b3 from '../../assets/BobaAssets/b3.svg';
+import b4 from '../../assets/BobaAssets/b4.svg';
+import b5 from '../../assets/BobaAssets/b5.svg';
+
+import Level from '../Levels';
+import '../styles/Lucy.scss';
+
+export default function BobaShop(): JSX.Element {
+  const [ isCompleted, setIsCompleted ] = useState(false);
+  const [password, setPassword] = useState('');
+
+  const complete = () => {
+    setIsCompleted(true);
+  };
+
+  const handleSubmit = (evt: { preventDefault: () => void }) => {
+    evt.preventDefault();
+    if (password == 'decaf') {
```

Comment:
do we want to put the password in firebase?

Reply:
Ah dang spoiler alert, i should have solved the puzzle first before reading the comments :((